src/core/public/i18n/i18n_service.mock.ts

```diff
@@ -16,11 +16,16 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
+import React from 'react';
 import { I18nService, I18nSetup } from './i18n_service';
 
+const PassThroughComponet = ({ children }: { children: React.ReactNode }) => children;
+
 const createSetupContractMock = () => {
   const setupContract: jest.Mocked<I18nSetup> = {
-    Context: jest.fn(),
+    // By default mock the Context component so it simply renders all children
+    Context: jest.fn().mockImplementation(PassThroughComponet),
```

Comment:
Since we might use that mock for testing other methods, I think it makes more sense having a default mock implementation here, that simply renders the children, so we don't need to specify that implementation in all other core services we want to use that mock during testing.